app/src/main/java/com/example/android/tvleanback/ui/MainFragment.java

```diff
@@ -84,8 +85,8 @@
     private Map<Integer, CursorObjectAdapter> mVideoCursorAdapters;
 
     @Override
-    public void onAttach(Context context) {
-        super.onAttach(context);
+    public void onAttach(Activity activity) {
+        super.onAttach(activity);
```

Comment:
alternative approach would be to override both and use if (Util.SDK_INT > 23) like in PlaybackFragment but this looks better